diff_cover/violations_reporter.py

```diff
@@ -116,14 +116,14 @@ def _get_classes(self, xml_document, src_path):
              src_abs_path in [
                  os.path.join(
                      source,
-                     clazz.get('filename')
+                     os.path.normpath(clazz.get('filename').replace("\\", "/"))
                  ) for source in sources]]
             or
             [clazz for clazz in classes if
-             clazz.get('filename') == src_abs_path]
+             os.path.normpath(clazz.get('filename').replace("\\", "/")) == src_abs_path]
             or
             [clazz for clazz in classes if
-             clazz.get('filename') == src_rel_path]
+             os.path.normpath(clazz.get('filename').replace("\\", "/")) == src_rel_path]
```

Comment:
No problem. One last thing to be careful of - `src_abs_path` and `src_rel_path` are both going to come out with some backslashes on Windows, so they will need altering too.